### PR TITLE
Development2

### PR DESCRIPTION
**Pull request by Yilikal250**
These changes complete the rest of the portfolio .This changes includes new HTML and CSS codes with new images. I have used inters on my Vs c to implement and see the correctness and errors of the changes i have made.
And before merging the files can you give me a feed back.

### CODE REVIEW

**Review by zainsadaqat — CHANGES_REQUESTED** [review]
# Hey @Yilikal250 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

Invalid Code Review Request ❎
You have submitted a project, that linter is not properly configured. Follow [this instruction](https://github.com/microverseinc/linters-config/tree/master/html-css#html--css3-course) to configure the linter properly. In your case [workflow](https://github.com/MaganaAsiati/Portofolio/tree/portofolio_skeleton/.github) should have to be a folder inside the .github file. Kindly update that for the next review.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.

Invalid Code Review Request Does not count into the code reviews limit.

File: index.html [hunk 1]
```diff
@@ -2,229 +2,220 @@
...
+  </section>
+  <footer id="Contact-me">
+    <p>
+      I'm always interested in hearing about new projects, so if you'd like to
+      chat please get in touch.
+    </p>
+    <form action="https://formspree.io/f/meqnlzqp" method="POST">
+      <input type="text" placeholder="Full Name" maxlength="30" required />
+      <input type="text" placeholder="Email Address" required />
+      <textarea name="textarea" placeholder="Enter text here" cols="30" rows="10" maxlength="500" required></textarea>
```
Comment (codepantha) [inline index.html]: - [ ] Please add a `type=email` attribute to your email input field in order to have HTML validation 🙏 
- [ ] You could add a `margin-bottom` to each input field in order to space it out and look a bit more like the figma design 😎

**Review by codepantha — CHANGES_REQUESTED** [review]
Hi @Yilikal250 👋 , 

# STATUS: CHANGES REQUESTED ♻️ 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


### To Highlight 💡 
- Meaningful commits ✅ 
- Good UI design ✅ 
- Descriptive PR ✅ 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:

- [ ] You can consider placing all your project's assets in an assets folder to declutter your application 😎 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Amrhub — APPROVED** [review]
# Approved 💯 ✅ 🥳 🎈 🎉 

Hi @Yilikal250, 👋🏻 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights ✅ 
Highly organized and clean code ✔️ 👨🏻‍💻 
Professional documentation ✔️ 
Correct workflow ✔️ 
Meaningful commit messages ✔️ 


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**